templates/patroni.yml.j2

```diff
@@ -24,7 +25,6 @@ bootstrap:
   - auth-host: md5
   - auth-local: trust
   - encoding: UTF8
-  - locale: en_US.UTF-8
```

Comment:
Setting a locale errors out, likely not generated in the image. The VM charm does not set it.